Frameworks/GLKit/GLKTexture.mm

```diff
@@ -274,18 +275,26 @@ @implementation GLKTextureLoader {
 }
 
 /**
- @Status Interoperable
+* Creates an image from file, if the image is not a GUID_WICPixelFormat32bppPRGBA format, it is converted to
+* GUID_WICPixelFormat32bppPRGBA and returned.
 */
-+ (GLKTextureInfo*)textureWithContentsOfFile:(NSString*)fname options:(NSDictionary*)opts error:(NSError**)err {
-    CGDataProviderRef provider = CGDataProviderCreateWithFilename([fname UTF8String]);
-    CGImageRef img = _CGImageCreateFromDataProvider(provider);
+static inline CGImageRef __Create32bppPRGBAImageFromFile(NSString* fname) {
```

Comment:
nit: if you do make this API, please perhaps call it `_CGImageCreateFromFileWithWICFormat` -- specifically, we need to differentiate it from the CGBitmapInfo/CGAlphaInfo APIs.